### ISSUE TITLE
plotProfile() x-label colour for N2

### ISSUE THREAD

**Issue by clayton33** [dankelley/oce#927]
See example

```
library(oce)
data(ctd)
plotProfile(ctd, xtype='N2', col='blue')
```

**Comment by richardsc** [comment]
What is the issue? Is it that the axis label colour for N2 also gets coloured blue?

I suspect there's possibly some conflict here between the fact that there is a specific `col.N2` argument (because of the `xtype="density+N2"` option) and the behaviour when a single field is plotted.

**Comment by clayton33** [comment]
Yes, issue is that the axis label is coloured with 'col' argument, but the line should just be coloured blue.

**Comment by dankelley** [comment]
I think this and #925 are both fixed in commit f174db42cd74ae798a559f2fd27a1198fd968275 of the `develop` branch.
